llama.cpp

```diff
@@ -2359,7 +2359,7 @@ static bool llama_kv_cache_init(
     cache.recurrent = model.arch == LLM_ARCH_MAMBA;
     cache.v_trans   = !cparams.flash_attn;
 
-    // TODO: support mixed reccurent Transformer architectues
+    // TODO: support mixed reccurent Transformer architectures
```

Comment:
While we're fixing this comment:
```suggestion
    // TODO: support mixed recurrent Transformer architectures
```

Reply:
Thanks, If you have pull request, I will close this pr. If not I will fix `reccurent` -> `recurrent` together.